packages/ember-application/lib/system/application.js

```diff
@@ -645,24 +795,58 @@ var Application = Namespace.extend(RegistryProxy, {
     @method didBecomeReady
   */
   didBecomeReady() {
-    if (this.autoboot) {
-      this.runInstanceInitializers(this.__deprecatedInstance__);
-
-      if (environment.hasDOM) {
-        this.__deprecatedInstance__.setupEventDispatcher();
-      }
-
-      this.ready(); // user hook
-      this.__deprecatedInstance__.startRouting();
-
+    try {
+      // TODO: Is this still needed for _globalsMode = false?
       if (!Ember.testing) {
```

Comment:
I *think* the two are related, from what I saw in tests `BOOTED` basically means "did we process the entire namespace"? I'm.. 70% sure :P